torchvision/models/detection/ssd.py

```diff
@@ -30,10 +30,7 @@ class SSD300_VGG16_Weights(WeightsEnum):
         url="https://download.pytorch.org/models/ssd300_vgg16_coco-b556d3b4.pth",
         transforms=ObjectDetection,
         meta={
-            "task": "image_object_detection",
-            "architecture": "SSD",
             "num_params": 35641826,
-            "size": (300, 300),
```

Comment:
Q: Do the detection models have a `min_size` too?

Reply:
Good question. No they don't because they do resize within the model. As a result even if you give them a (1, 1) size image, it will end up being somewhere between 800 and 1333 pixels. Depending on the ratios you can get with sizes in between, making it hard to define what the size actually is.